src/NHSD.BuyingCatalogue.Solutions.API/Controllers/BrowserBased/BrowsersSupportedController.cs

```diff
@@ -55,7 +55,7 @@ public async Task<ActionResult> GetBrowsersSupportedAsync([FromRoute][Required]s
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
         public async Task<ActionResult> UpdateBrowsersSupportedAsync([FromRoute] [Required] string id,
             [FromBody] [Required] UpdateBrowserBasedBrowsersSupportedViewModel viewModel) =>
-            (await _mediator.Send(new UpdateSolutionBrowsersSupportedCommand(id, viewModel)).ConfigureAwait(false))
+            (await _mediator.Send(new UpdateSolutionBrowsersSupportedCommand(id, viewModel?.Trim())).ConfigureAwait(false))
```

Comment:
If we are applying the trim here, could we leave the implementation on the view model. Meaning we wouldn't need the extension method.